ogre_mesh_viewer.py

```diff
@@ -200,6 +201,21 @@ def draw_loading(self):
         ImGui.Text("Loading..            ")
         ImGui.End()
 
+    def reload(self):
+        if app.infile:
+            app.restart = True
+            app.getRoot().queueEndRendering()
+
+    def load_file(self):
+        # Avoid recursive calling, which might block the window manager
```

Comment:
The recursive calling?

I put that there (the lock) because you said that you managed to both the entire window manager by opening the file dialog twice if I remember correctly.